src/Microsoft.AspNetCore.OData/Extensions/SerializableErrorExtensions.cs

```diff
@@ -117,7 +176,11 @@ private static TValue GetPropertyValue<TValue>(this SerializableError error, str
         }
     }
 
-    internal static class SerializableErrorKeys
+    /// <summary>
+    ///     Different keys for adding entries to an <see cref="SerializableError" /> instance so
+    ///     that it can be parsed to a <see cref="ODataError" /> instance
+    /// </summary>
+    public static class SerializableErrorKeys
```

Comment:
This bit of the change is causing a test failure for the PublicApi UT since this is now public. Does it need to be public? If it does, you'll need to run the TestPublicApi UT and update the baseline (.bsl file). I think the instructions are wrong at the moment and there are 2 AspNetCore .bsl files, you'll need to update the one that is not in the shared directory. The UTs all run and pass in VS.

Reply:
The test should be fine, I have it running in my latest PR. But like I said, the instructions are wrong and tell you to update the bsl file in the shared test dir. The bsl in the AspNetCore directory is the one that is actually used so you'll need to update that one.

Reply:
Fixed! Also added a fix to correct the wrong copy instruction.

Reply:
Looks like the public API is still failing in VSTS, did it pass locally?